app/insights.html

```diff
@@ -0,0 +1,145 @@
+  <script>
+    (function (i, s, o, g, r, a, m) {
+      i['GoogleAnalyticsObject'] = r;
+      i[r] = i[r] || function () {
+          (i[r].q = i[r].q || []).push(arguments)
+        }, i[r].l = 1 * new Date();
+      a = s.createElement(o),
+        m = s.getElementsByTagName(o)[0];
+      a.async = 1;
+      a.src = g;
+      m.parentNode.insertBefore(a, m)
+    })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
+
+    ga('create', 'UA-73481201-1', 'auto');
+    ga('send', 'pageview');
+
+  </script>
+
+  <style>
+    .card{
+      margin-bottom: 70px;
+    }
+  </style>
+
+</head>
+<body style="padding-top: 70px;">
+
+<nav id="mainNav" class="navbar navbar-default navbar-fixed-top">
+  <div class="container-fluid">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
+              data-target="#bs-example-navbar-collapse-1">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand page-scroll" href="index.html#page-top">♥ Treasury</a>
+    </div>
+
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+      <ul class="nav navbar-nav navbar-right">
+        <li>
+          <a class="page-scroll" href="journal.html">Journal</a>
+        </li>
+        <li>
+          <a class="page-scroll" href="index.html#about">About</a>
+        </li>
+        <li>
+          <a class="page-scroll" href="index.html#services">Services</a>
+        </li>
+        <li>
+          <a class="page-scroll" href="index.html#privacy">Privacy</a>
+        </li>
+        <li>
+          <a class="page-scroll" href="index.html#contact">Contact</a>
+        </li>
+        <li>
+          <a class="page-scroll" id="profile_link" href="Account.html">Account</a>
+        </li>
+        <li>
+          <a class="page-scroll" id="logout_link" style="display: none" href="logout.html">Logout</a>
+        </li>
+      </ul>
+    </div>
+    <!-- /.navbar-collapse -->
+  </div>
+  <!-- /.container-fluid -->
+</nav>
+
+<div class="wrapper">
+  <div class="container">
+
+    <div class="masonry-container">
+
+      <div id="insights"></div>
+
+    </div>
+  </div>
+</div>
+
+<script type="text/javascript">
+  const endpoint = 'http://localhost:3000';
+  const form = document.getElementById("new-article");
+  fetch(`${endpoint}/insights`, {
+    method: 'get',
+    headers: {
+      'Accept': 'application/vnd.api+json',
+      'Content-Type': 'application/vnd.api+json'
+    }
+  }).then(function(response) {
+    return response.json()
+  }).then(function(json) {
+    console.log('parsed json', json.data);
+    for (x in json.data) {
```

Comment:
@DonnieWest 
Is there a better way of handling the json response and enumerating through it?